spec/requests/api/conditions_spec.rb

```diff
@@ -18,6 +18,91 @@ def assign_conditions_to(resource)
     resource.conditions = Condition.all
   end
 
+  context "Condition CRUD" do
+    let(:sample_condition) do
+      {
+        :name        => "name",
+        :description => "description",
+        :expression  => {"=" => {"field" => "ContainerImage-architecture", "value" => "dsa"}},
+        :towhat      => "ExtManagementSystem",
+        :modifier    => "allow"
+      }
+    end
+    let(:condition) { FactoryGirl.create(:condition) }
+    let(:condition_url) { conditions_url(condition.id) }
+    let(:conditions) { FactoryGirl.create_list(:condition, 2) }
+
+    it "forbids access to conditions without an appropriate role" do
```

Comment:
I don't think this test is related to the code you're submitting here. Can you add tests for authorization only on the actions you've added? Thanks! (Though you are welcome to add test coverage if none exists in a separate PR)